mkosi/resources/mkosi-obs/mkosi.build

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Second stage of build:
+# - signed hashes are in cpio archive in SOURCES/ together with artifacts from previous build
+# - attach the signatures to the UKI(s) with pesign
+# - remove shasums of previous artifacts, given we are re-creating them after this step
+# - place artifacts from previous builds and signed UKI in output directory
+
+set -e
+set -x
+
+if [ ! -f /usr/src/packages/SOURCES/mkosi.cpio.rsasign.sig ]; then
+    exit 0
+fi
+
+cpio -t < /usr/src/packages/SOURCES/mkosi.cpio.rsasign.sig
```

Comment:
Nit here and elsewhere to be consistent with the style in the systemd repo
```suggestion
cpio -t </usr/src/packages/SOURCES/mkosi.cpio.rsasign.sig
```
Same for output redirection `>foo` instead of `> foo`.